buildpacks/nodejs/bin/build

```diff
@@ -22,16 +22,22 @@ rm -rf node_modules
 
 clear_cache_on_stack_change "$layers_dir"
 
+write_to_store_toml "$layers_dir"
+
 set_up_environment "$layers_dir/nodejs"
 
 bootstrap_buildpack "$layers_dir/bootstrap"
 
 install_or_reuse_toolbox "$layers_dir/toolbox"
 export PATH=$layers_dir/toolbox/bin:$PATH
 
+store_node_version "$layers_dir/nodejs"
```

Comment:
If I run `clear_cache_on_node_version_change` before `install_or_reuse_node` I get a `node: command not found` for the `node -v` cmd in the `clear_cache_on_node_version_change` because node has not been installed yet.